front-end/healthcheckstub/server.go

```diff
@@ -0,0 +1,40 @@
+// MIT License. Copyright (c) 2020 CQFN
+// https://github.com/cqfn/degitx/blob/master/LICENSE
+
+// Package healthcheckstub emulates healthcheck
+// to allow gitlab to start with degitx that has unimplemented methods.
+package healthcheckstub
+
+import (
+	"context"
+
+	"cqfn.org/degitx/logging"
+	healthpb "google.golang.org/grpc/health/grpc_health_v1"
+)
+
+type server struct {
+	log *logging.Logger
+}
+
+func NewServer(log *logging.Logger) healthpb.HealthServer {
+	return &server{
+		log: log,
+	}
+}
+
+func (s server) Check(
+	_ context.Context,
+	_ *healthpb.HealthCheckRequest,
+) (*healthpb.HealthCheckResponse, error) {
+	return &healthpb.HealthCheckResponse{
```

Comment:
@stepanvalyavskiy and here we need a task to implement health check correctly (return `SERVING` only if checks passed)